src/qt/pivx/send.cpp

```diff
@@ -249,22 +249,25 @@ void SendWidget::clearEntries(){
     addEntry();
 }
 
-void SendWidget::addEntry(){
-    if(entries.isEmpty()){
-        createEntry();
+void SendWidget::addEntry() {
+    if (entries.isEmpty()) {
+        SendMultiRow *sendMultiRow = createEntry();
+        sendMultiRow->setFocus();
```

Comment:
You can set focus at the bottom of the method and do it only once for both paths of the if/else statement.